### DISCUSSION TITLE
Sponsor Block

### DISCUSSION THREAD

**Discussion by JhonDesayuna** [fast4x/RiMusic#3286] [Ideas]
Hello, I have an idea that might be good. I've been using the "SponsorBlock" extension for a while, and it's great for music because (besides removing ads within videos) it removes all the silent and non-musical parts (like noises or sounds from the music video).

As you can see in the image, all the sections without music are marked in orange, and it skips them. For example, this song would remove a part in the middle that kind of kills the vibe.
![image](https://github.com/user-attachments/assets/bc2686ba-1797-472a-be7a-58a7699a64bd)

 It would be amazing to include it within the app. Thank you very much for the app you have created, and best regards.

**Comment by fast4x** [comment]
Hi, this will be added when RiMusic will have support for videos.

**Reply by JhonDesayuna** [reply]
Hello, thank you very much for your response. I'm glad to hear you have such great ideas to implement in the future. I’m looking forward to it. Best regards, and I hope it arrives soon 😉😊.
